libsolidity/inlineasm/AsmParser.cpp

```diff
@@ -95,7 +95,8 @@ assembly::Statement Parser::parseStatement()
 			fatalParserError("Label name / variable name must precede \":\".");
 		assembly::Identifier const& identifier = boost::get<assembly::Identifier>(statement);
 		m_scanner->next();
-		if (m_scanner->currentToken() == Token::Assign)
+		// =: is an assignment, while :=: is invalid
```

Comment:
Please change the comment:
`identifier:=:` should be parsed as `identifier: =:` (i.e. a label) while `identifier:=` (being followed by a non-colon) as `identifier :=` (assignment).